README.md

```diff
@@ -225,6 +224,15 @@ $ aac --help
 # For example, to validate the spec.yaml file
 $ aac validate src/aac/spec/spec.yaml
 ```
+
+#### Running on MacOS
```

Comment:
Based on the changes to `setup.py` that @Coffee2Bits has in #130, we may not need this.